www/front_src/src/route-components/pollerWizard/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { lazy, useState, Suspense } from 'react';
 
 import { equals, isNil, path } from 'ramda';
+import { Responsive } from '@visx/visx';
```

Comment:
Why use Visx `Responsive` when there is no graph?

Reply:
Responsive provides components to make your layout resposive to your parent element to prevent some mess up